check-required.sh

```diff
@@ -0,0 +1,21 @@
+#!/bin/bash
+
+# Exit immediately, but don't expose any environment variable values
+set +x
+set -e
```

Comment:
@danielbachhuber I'm going to move the `whoami` check to the bottom of the file. I can imagine someone running `test.sh` locally. They would not need to be logged into to Terminus but they would need the other four globabl variables.